blocks/shared-styles/_children/headings/heading/index.jsx

```diff
@@ -12,7 +12,8 @@ const Heading = (props) => (
 	<LevelContext.Consumer>
```

Comment:
Any reason for these changes? We are trying to remove the usage of these components

Reply:
As mentioned in the huddle, this change was mainly due to the linter flagging these files for review and it coming up.